swift/ql/test/library-tests/dataflow/dataflow/DataFlowInline.expected

```diff
@@ -1,2 +1,3 @@
-failures
 testFailures
+| test.swift:863:24:864:1 | // $ flow=873\n | Missing result: flow=873 |
```

Comment:
Iterating through varargs, where the non-iterating reads of the same varargs still have flow.
```
sink(arg: args[0]) // $ SPURIOUS: flow=873
sink(arg: args[1]) // $ flow=873

for arg in args {
    sink(arg: arg) // $ flow=873 <<< NOW MISSING
}
```
It looks like we've lost some flow through `for arg in args`.

Reply:
Fixed by https://github.com/github/codeql/pull/18143 .